docs/DeveloperGuide.adoc

```diff
@@ -279,51 +279,203 @@ Refer to the guide <<DevOps#, here>>.
 * prefer desktop apps over other types
 * can type fast
+* reads and writes competently in English
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: convenient all-in-one app for effectively managing diabetes that is faster than a typical mouse/GUI driven app
 
 [appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="90%",cols="20%,<25%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |diabetic patient who has different options on medical care |know exactly how much I am spending on medication and consultation |know which hospitals to seek medical care from
 
-|`* * *` |user |add a new person |
+|`* * *` |very busy diabetic |use a flexible calendar system that can account for updates | easily make changes to appointments that I have to change often due to other commitments
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |diabetic |keep track of my medical expenses |better manage my finance
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |person who likes numbers |see summary statistics |better track my progress
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |diabetic |get an overview of my dieting/exercising data regularly |save time because I am working 9-5
+
+
+|`* * *` |forgetful diabetic |be reminded to attend my medical appointments |know how well my existing measures work
+
+|`* * *` |patient who has recently been diagnosed of diabetes |be informed when I eat food with high sugar content |live better and reduce the chances of further health deterioration
+
+|`* * *` |lazy diabetic |have reminders for exercising |force myself to work out.
+
+|`* * *` |busy diabetic |be reminded on when to refill / stock up on insulin|
+
+|`* * *` |diabetic |see graphic data summary |minimise the need to read long paragraphs
+
+|`* * *` |diabetic patient who has just been recently diagnosed |have some motivation and reminders on my diet |reduce my struggles of cutting down on meals or even exercise that is really tough for me
+
+|`* * *` |diabetic |automatically calculate my daily sugar/carb intake |eliminate the trouble to search for the levels of sugar content in the food I eat everyday.
```

Comment:
similar to line 321. (duplicated user story), if you expect to be informed about sugar content, we can expect that the daily sugar/carb intake has been calculated.